src/components/Template.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export interface Props {
+  readonly children: string;
+  readonly slot: string;
+}
+
+export function Template({ children, ...attrs }: Props): JSX.Element {
+  return <template {...attrs} dangerouslySetInnerHTML={{ __html: children }} />;
+}
```

Comment:
Pretty sure this should work if you want to stick to JSX.

```tsx
import React from 'react';
import { renderToString } from 'react-dom';

export interface Props {
  readonly children: React.ReactNode | React.ReactNode[];
  readonly slot: string;
}

export function Template({ children, ...attrs }: Props): JSX.Element {
  return <template {...attrs}>{renderToString(<>{children}</>)}</template>;
}
```

Reply:
This doesn't allow me to do stuff like this, `<div style="{{row.data.style}}"></div>`